doc/reference/reference_lua/config/utils_schema.rst

```diff
@@ -0,0 +1,1251 @@
+    :param string raw_value: environment variable value
+    :param schema_node_object schema_node: a schema node (see :ref:`schema_node_object <config-utils-schema_node_object>`)
+
+    :return: the parsed value
+    :rtype: table
+
+    **See also:** :ref:`config_utils_schema_env-vars`
+
+..  _config-utils-schema-map:
+
+..  function:: schema.map(map_def)
+
+    Create a map node of a configuration schema.
+
+    :param table map_def: a table in the following format:
+
+                            ..  code-block:: text
+
+                                {
+                                    key = <schema_node_object>,
+                                    value = <schema_node_object>,
+                                    <..annotations..>
+                                }
+
+                          See also: :ref:`schema_node_object <config-utils-schema_node_object>`, :ref:`schema_node_annotation <config-utils-schema_node_annotation>`.
+
+    :return: the created map node as a table with the following fields:
+
+            -   ``type``: ``map``
+            -   ``key``: map key type
+            -   ``value``: map value type
+            -   annotations, if provided in ``map_def``
+
+    :rtype: table
+
+    **See also:** :ref:`config_utils_schema_nodes_map`
+
+..  _config-utils-schema-new:
+
+..  function:: schema.new(schema_name, schema_node[, { methods = <...> }])
+
+    Create a schema object.
+
+    :param string schema_name: a name
+    :param table schema_node: a root schema node
+    :param table methods: methods
+
+    :return: a new schema object (see :ref:`schema_object <config-utils-schema_object>`)
+             as a table with the following fields:
+
+                *   ``name``: the schema name
+                *   ``schema``: a table with schema nodes
+                *   ``methods``: a table with user-provided methods
+
+    :rtype: table
+
+    **See also:** :ref:`config_utils_schema_getting_started`
+
+..  _config-utils-schema-record:
+
+..  function:: schema.record(fields[, annotations])
+
+    Create a record node of a configuration schema.
+
+    :param table fields: a table of fields in the following format:
+
+                         ..  code-block:: text
+
+                             {
+                                 [<field_name>] = <schema_node_object>,
+                                 <...>
+                             }
+
+                         See also: :ref:`schema_node_object <config-utils-schema_node_object>`.
+
+    :param table annotations: annotations (see :ref:`config_utils_schema_annotation`)
+
+    :return: the created record node as a table with the following fields:
+
+            -   ``type``: ``record``
+            -   ``fields``: a table describing the record fields
+            -   annotations, if provided
+
+    :rtype: table
+
+    **See also:** :ref:`config_utils_schema_nodes_record`
+
+..  _config-utils-schema-scalar:
+
+..  function:: schema.scalar(type[, annotations])
```

Comment:
It is called as follows.

```lua
schema.scalar({
    type = <...>,
    <..annotations..>
})
```

It seems the prototype is incorrect.

Reply:
Thanks, fixed.